reco_utils/evaluation/python_evaluation.py

```diff
@@ -95,30 +102,25 @@ def merge_rating_true_pred(
 
     Returns:
         pd.DataFrame: Merged pd.DataFrame
+        str: Column with the true ratings
+        str: Column with the predicted ratings
     """
-    # Select the columns needed for evaluations
-    rating_true = rating_true[[col_user, col_item, col_rating]]
-    rating_pred = rating_pred[[col_user, col_item, col_prediction]]
-
-    if col_rating == col_prediction:
-        rating_true_pred = pd.merge(
+    suffixes = ["_true", "_pred"]
+    # Apart from merging both dataframes, pd.merge will rename the columns with the suffixes only if the rating
+    # column name of rating_true is the same as the name rating column name in rating_pred
+    rating_true_pred = pd.merge(
```

Comment:
yes, this is similar to what I did in #699, nice to know the suffixes are only added as needed, at that point it's probably not necessary to even create a separate variable for them.

I think we should go further though, there's nothing else in the merged dataframe that is important other than the truth and prediction columns, so I just extracted them and passed them back as two pd.Series. That way we don't need to keep track of the name of the columns.

Reply:
oh that's a great suggestion and make the code cleaner, I'll make the changes

Reply:
fixed